pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -103,7 +105,7 @@ def assert_frame_equal(self, left, right, *args, **kwargs):
         decimals = (left.dtypes == "decimal").index
 
         for col in decimals:
-            self.assert_series_equal(left[col], right[col], *args, **kwargs)
+            BaseDecimal.assert_series_equal(left[col], right[col], *args, **kwargs)
```

Comment:
this looks like it should be a classmethod

Reply:
What would be the advantage of using `classmethod` over `staticmethod`? I typically only use `classmethod` if I need multiple ways of instantiation

Reply:
having BaseDecimal hard-coded here looks like a kludge, I'm speculating that what you want here is `cls.assert_series_equal`, kind of halfway inbetween the regular method it is now and the staticmethod in the PR